src/footer/footer.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+import { Grid } from '@material-ui/core'
+
+import { Footergrid } from 'footer/footergrid'
+import { useStyles } from 'footer/styles'
+
+export const Footer = () => <footer className={useStyles().footer}>
+  <Grid className={useStyles().gridHeight} maxWidth={'lg'} container alignContent={'center'} justify={'center'}>
```

Comment:
Is this container really necessary? Its only child is another container, so something smells off. Let's maybe use a `<Box/>` component here, if really necessary, as it might suit your purposes better.

Check it out [here](https://material-ui.com/components/box/)

Reply:
Done